addons/portal/views/portal_templates.xml

```diff
@@ -351,92 +381,231 @@
         </div>
     </template>
 
+
+    <template id="address_kanban" name="Kanban address">
```

Comment:
Same remark here, template names should be prefixed and namespaced, the other ones all seem to start with "portal_".